src/ui/public/vis/editors/default/agg.html

```diff
@@ -3,8 +3,10 @@
 
   <!-- open/close editor -->
   <button
-    aria-label="{{ editorOpen ? 'Close Editor' : 'Open Editor' }}"
+    aria-label="Toggle {{agg.schema.title}} editor"
```

Comment:
this breaks a gauge chart selenium test, can you update the test to look for this new aria label or find this button in different way ?